### DISCUSSION TITLE
Which slot is the mesh saved in? Also a question about 3 point leveling.

### DISCUSSION THREAD

**Discussion by rocket59** [jneilliii/OctoPrint-BedLevelVisualizer#513] [Q&A] [answered]
So when I create a new mesh via the plugin and do an M500 does that save the new mesh to Slot 1, overwriting what's already there? Can a different slot be specified?

Also I see in some examples around the web that a G29 J is executed after loading a new mesh (within the cura start G code settings). I'm not sure what's the point of this if I've already loaded a recently created mesh. Wouldn't the G29 J interefere with the mesh settings?

**Comment by jneilliii** [comment, marked as answer]
I'm pretty sure if you don't specify a slot (assuming you're referring to UBL) it gets saved to slot 1, Otherwise you have to use M29 S# or M420 S# to save to a specific slot. 

I've never used the j option, but the idea is that the mesh is accounting for the bed variance, and then the j is accounting for any changes related to your springs loosening, etc. so it is orbiting the mesh to match the 3 point plane.

**Reply by rocket59** [reply]
Thanks!
